lobby/config/lobby/lobby.properties

```diff
@@ -1,21 +0,0 @@
-## Lobby server
```

Comment:
I believe, with the removal of the properties file, you'll also need to remove the following lines from the `lobby` project buildscript:

https://github.com/triplea-game/triplea/blob/254ac44ebb8b8e6a64bcd838b516ecd044ffba76/lobby/build.gradle#L26-L28

Reply:
I suppose this is reason/argument to have the PR build simulate the creation of artifacts, perhaps it should create artifacts and do some testing that the game/lobby can be launched.

Regardless, this is a good cleanup, updated: 03287ebf5388c00c43bca6d9e5aab5fa50bcc05a